src/amo/components/Footer/styles.scss

```diff
@@ -0,0 +1,49 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+.Footer {
+  background: $footer-base-color;
+  color: $white;
+  padding: 20px;
+
+  @include respond-to(medium) {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+.Footer-language-and-links {
+  max-width: 220px;
```

Comment:
This is pretty protective against screens that are lower than `220px`... realistically we could probably just make this `width: 220px` as that's what's in the desktop mocks and fits on even phones smaller than `280px` wide. Let me know.